### PR TITLE
New home screen

### LINKED ISSUE #35: [UI/UX] Make new home screen
Update MainFragment to reflect Home screen from the design: https://www.figma.com/file/0qkRp3TrKBRnpEnmQeA5m4/Covid-Watch---App-v1.6?node-id=36%3A394

We are in a hurry for an MVP so focus on just getting the basic look without the messages at the top for now.

Child of #23

### PR DESCRIPTION
**Pull request by jamesjmtaylor**
https://github.com/covid19risk/covidwatch-android/issues/35
Nesting this view in a scrollview was unavoidable due to the double buttons.  I placed the buttons inside their own relative layout so that they would remain on the bottom of the screen for larger devices (such as tablets).  There are several text strings that I added that are not currently used, but will be needed when we need to inform the user that they've been in contact with a COVID-19 positive individual.  Finally, I persisted the initial view boolean to shared preferences so that we could show the appropriate copy for subsequent visits.

allSet
share
welcomeBack

### CODE REVIEW

File: app/src/main/java/org/covidwatch/android/CovidWatchApplication.kt
Comment (madim) [inline app/src/main/java/org/covidwatch/android/CovidWatchApplication.kt]: I'm not sure if this needs to be launched in mainScope, because Application.onCreate() is already running in main thread.

File: app/src/main/java/org/covidwatch/android/ui/MainFragment.kt
Comment (madim) [inline app/src/main/java/org/covidwatch/android/ui/MainFragment.kt]: let's initialize it in `onCreate()` and get rid of all `?.`, `?: return` code :)

 ```kotlin
private lateinit var preferences : SharedPreferences

override fun onCreate(savedInstanceState: Bundle?) {
    preferences = requireContext().getPreferences(Context.MODE_PRIVATE)
}
```

File: app/src/main/java/org/covidwatch/android/ui/MainFragment.kt
Comment (madim) [inline app/src/main/java/org/covidwatch/android/ui/MainFragment.kt]: it's better to use `apply()` here to save value asynchronously

**Review by nkumarcc — APPROVED** [review]
Love the UI, hit the recommendations from madim. Also delete strings you deprecate as you go. Not blocking but if you can do/we do in the future will be dope.

I think at the least you should hit madim's suggestions, but accepting to unblock you to merge right after.

File: app/src/main/java/org/covidwatch/android/ui/MainFragment.kt
Comment (jamesjmtaylor) [reply app/src/main/java/org/covidwatch/android/ui/MainFragment.kt]: Done.

File: app/src/main/java/org/covidwatch/android/ui/MainFragment.kt
Comment (jamesjmtaylor) [reply app/src/main/java/org/covidwatch/android/ui/MainFragment.kt]: Done.

File: app/src/main/java/org/covidwatch/android/CovidWatchApplication.kt
Comment (jamesjmtaylor) [reply app/src/main/java/org/covidwatch/android/CovidWatchApplication.kt]: Removed.  Previously it was crashing on this line number complaining that Services must started on the main thread.  Looks like an unrelated change has fixed it, since it no longer crashes now that I've removed the the coroutine scope.

**Pull request merged**
